ai-ml/trainium-inferentia/addons.tf

```diff
@@ -218,23 +190,40 @@ module "eks_blueprints_addons" {
   tags = local.tags
 }
 
-resource "aws_eks_access_entry" "this" {
-  cluster_name  = module.eks.cluster_name
-  principal_arn = module.eks_blueprints_addons.karpenter.node_iam_role_arn
-  type          = "EC2_LINUX"
-}
-
 #---------------------------------------------------------------
 # Data on EKS Kubernetes Addons
 #---------------------------------------------------------------
 module "eks_data_addons" {
-  source  = "aws-ia/eks-data-addons/aws"
-  version = "1.33.0" # ensure to update this to the latest/desired version
+  #source  = "aws-ia/eks-data-addons/aws"
```

Comment:
Will fix this after https://github.com/aws-ia/terraform-aws-eks-data-addons/pull/46 is merged